test/server/services/LocationService.spec.js

```diff
@@ -0,0 +1,68 @@
+import sinon from 'sinon';
+import { expect } from 'chai';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+
+const LocationService = require('../../../server/services/location.js');
+let axiosMock;
+
+beforeEach(function(){
+    axiosMock = new MockAdapter(axios);
+});
+
+describe('LocationService', function() {
```

Comment:
Are these tests breaking just because of the dependency on the config file?  Seems like maybe.